FindPythonLibsNew.cmake

```diff
@@ -0,0 +1,281 @@
+# - Find python libraries
```

Comment:
Why do we need to vendor this file in? Isn't it part of cmake or pybind11?

Reply:
We should rely on upstream functionality as much as possible. If this script is only available in recent cmake versions, then it is better to restrict the cmake dependency to such a minimum version.

Reply:
It was discussed in https://github.com/impy-project/chromo/pull/82#issuecomment-1315408623

Reply:
But this should not be needed. I can build wheels for pyhepmc and iminuit fine without this.

Reply:
Hm. I will try to build without it, but I don't think that something is changed since the last time.

Reply:
Can be removed, in #126 I use standard cmake tools to find Python. This works correctly.

Reply:
I removed it, but I got an error on cibuildwheels. We should keep in mind, that cibuildwheel build the project in the containers `manylinux`.

> -- Found PythonInterp: /opt/python/cp38-cp38/bin/python (found suitable version "3.8.13", minimum required is "3.6")
> 2023-02-10T07:20:06.1773767Z   -- Found PythonLibs: python3.8
> 2023-02-10T07:20:06.1802501Z   -- Performing Test HAS_FLTO
> 2023-02-10T07:20:06.2783953Z   -- Performing Test HAS_FLTO - Success
> 2023-02-10T07:20:06.2835350Z   -- Could NOT find pybind11 (missing: pybind11_DIR)
> 2023-02-10T07:20:07.5507451Z   CMake Error at /tmp/pip-build-env-mz1x8pi0/overlay/lib/python3.8/site-packages/cmake/data/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
> 2023-02-10T07:20:07.5510907Z     Could NOT find Python (missing: Python_INCLUDE_DIRS Python_LIBRARIES
> 2023-02-10T07:20:07.5515302Z     Development Development.Module Development.Embed) (found version "2.6.6")
> 2023-02-10T07:20:07.5518962Z   Call Stack (most recent call first):
> 2023-02-10T07:20:07.5523247Z     /tmp/pip-build-env-mz1x8pi0/overlay/lib/python3.8/site-packages/cmake/data/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
> 2023-02-10T07:20:07.5526866Z     /tmp/pip-build-env-mz1x8pi0/overlay/lib/python3.8/site-packages/cmake/data/share/cmake-3.25/Modules/FindPython.cmake:561 (find_package_handle_standard_args)
> 2023-02-10T07:20:07.5530061Z     CMakeLists.txt:37 (find_package)
> 2023-02-10T07:20:07.5533357Z 
> 2023-02-10T07:20:07.5536719Z 
> 2023-02-10T07:20:07.5540708Z   -- Configuring incomplete, errors occurred!
> 2023-02-10T07:20:07.5544727Z   See also "/project/build/temp.linux-x86_64-cpython-38/CMakeFiles/CMakeOutput.log".
> 2023-02-10T07:20:07.5645635Z   Error: CMake configuration failed
> 2023-02-10T07:20:07.5954254Z   error: subprocess-exited-with-error
> 2023-02-10T07:20:07.5954528Z   
> 2023-02-10T07:20:07.5954917Z   × Building wheel for chromo (pyproject.toml) did not run successfully.
> 2023-02-10T07:20:07.5955246Z   │ exit code: 1
> 2023-02-10T07:20:07.5955502Z   ╰─> See above for output.

@HDembinski, do you have an idea how to fix it without returning to old version?

Reply:
Ok. The problem seems to be solved. I remove `REQUIRED` from `find_package(Python REQUIRED COMPONENTS Interpreter Development)`, i.e. changed to `find_package(Python COMPONENTS Interpreter Development)`

Reply:
`find_package(Python REQUIRED COMPONENTS Interpreter Development)` is incorrect. You need `find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)`. You do not need the `Development.Embed` module and it does not ship in the manylinux container.